public/data/praveenscience.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Praveen Kumar Purushothaman",
+  "bio": "Full Stack JavaScript Specialist",
+  "avatar": "https://i.imgur.com/wbUfjzdl.png",
```

It would be great if you can replace 
```suggestion
  "avatar": "https://github.com/praveenscience.png",
```

---

But why? I would like the other pic. The pic I have got here (https://i.imgur.com/wbUfjzdl.png) will be replacing the GitHub ones soon.